lib/yum/build.sh

```diff
@@ -40,6 +40,10 @@ if [ -f /etc/os-release ]; then
       # The ID is translated for repository layout
       distribution=amazon
       ;;
+    rocky)
+      # The ID is translated as transitional layout
+      distribution=centos
```

Comment:
I think it's not needed to keep compatibility.
I prefer using the name as is (rocky) to avoid confusion.

Reply:
Ok, just removed.